### PR TITLE
added architecture video

### PR DESCRIPTION
**Pull request by HPrinz**
Preview: 
```
bundle exec jekyll serve
```
Open http://127.0.0.1:4000

### CODE REVIEW

**Comment by lsund** [comment]
Hey Hanna! Very nice, I had something like this in mind the last time we talked about it. A simple graphic that pictures the basic idea of the service mesh in context and what it can do.
 One question though, It seems to show 4 microservices (the 4 red boxes) but two of them is called A and two called B. Is this a mistake or am I missing something? I would have expected Microservice A, Microservice B, Microservice C and Microservice D.

To make the picture more integrated with the text I'd somehow refer to the picture in the text, or add a caption that describes picture a bit. Also, there are traffic metrics, business metrics and just metrics. Maybe it makes sense to call it all metrics and let the user read the text  to find out more? 

The last two ideas are just suggestions though.

**Comment by HPrinz** [comment]
Thanks a lot! This feedback was very valuable! 
I saw that I was missing the information that the left side was separate form the right. The graphic was meant to show the same application, with and without a service mesh. The current solution should make this clear. I have also decided that a video is not the right type of media and have changed it to an image with a caption.  
Would you mind taking a look?

File: _includes/intro.html [hunk 1]
```diff
@@ -8,14 +8,30 @@ <h3> What is a Service Mesh? </h3>
...
-  <video width="100%" autoplay onclick="this.play()">
-    <source src="img/servicemesh.mp4" type="video/mp4">
-    Your browser does not support the video tag.
-  </video>
   <br/>
   <br/>
 
+  <figcaption >
+    <b>Without a Service Mesh</b>, each Microservice implements Business Logic and Cross Cutting Concerns (CCC) by itself. <br>
+   <b>With a Service Mesh</b>, Cross Cutting Concers (CCC) like traffic metrics, routing, and encryption are moved ouf the Microservice and into a proxy.
```
Comment (lsund) [inline _includes/intro.html]: Cross Cutting Concerns*

Comment (lsund) [reply _includes/intro.html]: moved out of ... ?

File: _includes/intro.html [hunk 2]
```diff
@@ -8,14 +8,30 @@ <h3> What is a Service Mesh? </h3>
...
-  </video>
   <br/>
   <br/>
 
+  <figcaption >
+    <b>Without a Service Mesh</b>, each Microservice implements Business Logic and Cross Cutting Concerns (CCC) by itself. <br>
+   <b>With a Service Mesh</b>, Cross Cutting Concers (CCC) like traffic metrics, routing, and encryption are moved ouf the Microservice and into a proxy. 
+    Business Logic and Business Metrics stay in the Microservice. Incoming and outgoing requests are transparently routed through the proxys. 
+    In addition to a layer of Proxys (Data Plane), a Service Mesh adds a so-called Control Plane. If distributes configuration updates to all proxys 
+    and receives metrics collected by the proxys for further processing, e.g. by monitoring infrastructure such as Prometheus.
```
Comment (lsund) [inline _includes/intro.html]: ... by a monitoring infrastructure ...

**Pull request merged**